test/kubeclient_test.rb

```diff
@@ -1,7 +1,4 @@
-require 'minitest/autorun'
-require 'json'
-require 'webmock/minitest'
-require './lib/kubeclient'
+require './test/test_helper'
```

Comment:
Interesting, so here's what I think is happening.  The `require` method doesn't actually work with relative paths.  That relative path is relative to your current working directory, so the reason is works is because you are running the tests from the root.  If you want to use relative paths from the file, use `require_relative`, which would make these `require_relative './test_helper'` or even, I think just `require_relative 'test_helper'`.

However, you shouldn't need all that.  The test directory should be on the load path, and you should just be able to do `require 'test_helper'` all by itself.  I think the reason it's not working is because of how the [test task is defined in the Rakefile](https://github.com/abonas/kubeclient/blob/master/Rakefile#L7-L9), which should be using a Rake::TestTask, like so:

```ruby
Rake::TestTask.new do |t|
  t.libs << "test"
end
```

Then in the test files you can just do `require 'test_helper'`